data/json/items/armor/undergarment.json

```diff
@@ -0,0 +1,743 @@
+    "symbol": "[",
+    "looks_like": "boxer_briefs",
+    "color": "light_red",
+    "covers": [ "LEGS" ],
+    "coverage": 15,
+    "material_thickness": 1,
+    "flags": [ "VARSIZE", "WATER_FRIENDLY", "SKINTIGHT" ]
+  },
+  {
+    "id": "hot_pants_fur",
+    "type": "ARMOR",
+    "name": "fur hot pants",
+    "name_plural": "fur hot pants",
+    "description": "A simple pair of fur short shorts.",
+    "weight": "118 g",
+    "volume": "250 ml",
+    "price": 5500,
+    "material": [ "fur" ],
+    "symbol": "[",
+    "looks_like": "hot_pants_leather",
+    "color": "brown",
+    "covers": [ "LEGS" ],
+    "coverage": 15,
+    "warmth": 10,
+    "material_thickness": 1,
+    "flags": [ "VARSIZE", "SKINTIGHT" ]
+  },
+  {
+    "id": "hot_pants_leather",
+    "type": "ARMOR",
+    "name": "leather hot pants",
+    "name_plural": "leather hot pants",
+    "description": "A simple pair of leather short shorts.",
+    "weight": "106 g",
+    "volume": "250 ml",
+    "price": 2500,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "looks_like": "hot_pants",
+    "color": "dark_gray",
+    "covers": [ "LEGS" ],
+    "coverage": 15,
+    "warmth": 5,
+    "material_thickness": 1,
+    "flags": [ "VARSIZE", "SKINTIGHT" ]
+  },
+  {
+    "id": "leg_warmers",
+    "type": "ARMOR",
+    "name": "pair of leg warmers",
+    "name_plural": "pairs of leg warmers",
+    "description": "Snug, soft cloth sleeves to keep your legs warm.",
+    "weight": "46 g",
+    "volume": "500 ml",
+    "price": 1000,
+    "material": [ "cotton" ],
+    "symbol": "[",
+    "looks_like": "leggings",
+    "color": "light_gray",
+    "covers": [ "LEGS" ],
+    "coverage": 60,
+    "encumbrance": 1,
+    "warmth": 20,
+    "material_thickness": 1,
+    "flags": [ "SKINTIGHT" ]
+  },
+  {
+    "id": "leg_warmers_f",
+    "type": "ARMOR",
+    "name": "pair of fur leggings",
+    "name_plural": "pairs of fur leggings",
+    "description": "Snug fur sleeves to keep your legs warm.  Simple yet effective since prehistoric times.",
+    "weight": "694 g",
+    "volume": "1500 ml",
+    "price": 1500,
+    "material": [ "fur" ],
+    "symbol": "[",
+    "looks_like": "leg_warmers",
+    "color": "brown",
+    "covers": [ "LEGS" ],
+    "coverage": 60,
+    "encumbrance": 3,
+    "warmth": 40,
+    "material_thickness": 2,
+    "flags": [ "VARSIZE" ]
+  },
+  {
+    "id": "leg_warmers_xl",
+    "type": "ARMOR",
+    "name": "pair of XL leg warmers",
+    "name_plural": "pairs of XL leg warmers",
+    "description": "Large, soft, snug cloth sleeves to keep your exotic anatomy warm.",
+    "weight": "50 g",
+    "volume": "750 ml",
+    "price": 20000,
+    "material": [ "cotton" ],
+    "symbol": "[",
+    "looks_like": "leg_warmers",
+    "color": "light_gray",
+    "covers": [ "LEGS" ],
+    "coverage": 60,
+    "encumbrance": 7,
+    "warmth": 20,
+    "material_thickness": 2,
+    "flags": [ "VARSIZE", "SKINTIGHT", "OVERSIZE" ]
+  },
+  {
+    "id": "leggings",
```

Comment:
Leggings probably don't belong in undergarments either, although personally I feel fashion could learn a thing or two from this.